src/modules/journal/journal.actions.ts

```diff
@@ -10,6 +11,10 @@ export const LOAD_JOURNAL_FAILURE = '[JournalEffects] Load Journal Failure';
 export const LOAD_JOURNAL_SILENT = '[JournalEffect] Load Journal Silent';
 export const LOAD_JOURNAL_SILENT_FAILURE = '[JournalEffect] Load Journal Silent Failure';
 
+export const LOAD_COMPLETED_TESTS = '[JoarnalEffect] Load Completed Tests';
+export const LOAD_COMPLETED_TESTS_SUCCESS = '[JoarnalEffect] Load Completed Tests Success';
+export const LOAD_COMPLETED_TESTS_FAILURE = '[JoarnalEffect] Load Completed Tests Failure';
```

Comment:
small spelling error, should be [JournalEffect] not [JoarnalEffect]

Reply:
corrected the spelling mistake